src/main/resources/static/schema.graphql

```diff
@@ -10,6 +10,7 @@ type Query {
     getLocation(region: RegionType!): Location
     # order
     getOrderByOrderNumber(orderNumber: String!): Order
+    showAllOrders(searchString: String, page: Int, size: Int, sort: [String]): OrderPageInfo
```

Comment:
I think we don't need `searchString `as argument.
![image](https://user-images.githubusercontent.com/35082985/203171042-c5c6b822-4bcc-4d61-aa14-1c60d20dcbe7.png)